docs/blog/newsletter-august-2017.rst

```diff
@@ -0,0 +1,84 @@
+
+See `The Art of Documentation and Readme.md<https://pronovix.com/api-docs-london-2017#ben>`_ from API the Docs London for more. Or, if you're heading to `Write the Docs Prague<http://www.writethedocs.org/conf/eu/2017/>`_, check out one of the README-related sessions!
+
+****************************************
+Accessible docs, colorblindness edition!
+****************************************
+
+A variety of questions come up when we try to improve the `accessibility of our documentation <https://developer.mozilla.org/en-US/docs/Learn/Accessibility/What_is_accessibility>`_. Is our link text descriptive enough? Do all our images have alt text? Is our navigation usable from a variety of device types? Are we using semantic markup to our advantage/the advantage of our readers? This month, another accessibility question floated to the surface: How do we make our docs friendly to colorblind readers? As always, the community had a bunch of great suggestions!
+
+Colorblindness can affect not only the effectiveness of our screenshots and other graphics, but also the visual experience of our documentation on the whole. For a primer on how to think about colorblindness in doc design, `this blog post (from a web designer's perspective) <http://www.zingdesign.com/web-design-guidelines-for-color-blind-users/>`_, is a good starting point. In general, we're looking for high-contrast, visual cues that aren't strictly color-dependant (texture, shape, etc.), and a color palette that looks tolerable to as many eyeballs as possible.
+
+For that last point, it can be hard to imagine what colors might look like to the various kinds of colorblindness. But this is the internet; there's tools for that! Folks suggested a couple of different options, including `Color Oracle <http://colororacle.org/>`_, a desktop simulator for the different types of colorblindness, and
+`Color Hexa <http://www.colorhexa.com/>`_, which provides a *wealth* of information about any color you look up, including how it appears for various types of colorblindness.
+
+************************
+Making docs maintainable
+************************
+This month the WTD hivemind tackled one of the holy grail questions: how do we establish a truly maintainable docs system? No surprise that we couldn't come up with a definitive set of practices, but folks contributed a wealth of great practical tips:
+* Implement well-structured information architecture
+* Automate as much as possible, especially in areas of rapid change, and make sure that automation scripts are easy to maintain
+*Examples: code includes, API reference docs, screenshots*
```

Comment:
Is this the right formatting? Does it indent properly within the list? (Sorry, being lazy and not looking up here :-( )